post-scan-actions/fss-to-securityhub/template.yaml

```diff
@@ -0,0 +1,106 @@
+Parameters:
+  snsScanResultTopicArn:
+    Type: String
+    Description: FSS Results SNS Topic ARN.
+  LambdaZipBucket:
+    Type: String
+    Description: S3 bucket for lambda zip. If your have it under s3://bucket/path/to/zip.zip, enter 'bucket' here. 
+  LambdaZipKey:
+    Type: String
+    Description: S3 key for lambda zip. If your have it under s3://bucket/path/to/zip.zip, enter 'path/to/zip.zip' here. 
+Resources:
+  sentToSecurityHubLambdaFunctionServiceRole58D0F596:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+        Version: "2012-10-17"
+      ManagedPolicyArns:
+        - Fn::Join:
+            - ""
+            - - "arn:"
+              - Ref: AWS::Partition
+              - :iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+  sentToSecurityHubLambdaFunctionServiceRoleDefaultPolicy9B55E2DF:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action: securityhub:BatchImportFindings
+            Effect: Allow
+            Resource:
+              Fn::Join:
+                - ""
+                - - "arn:aws:securityhub:"
```

Comment:
A missed `aws` -> `AWS::Partition`.

Reply:
🤔 This might cause issue for people that deploy this in a different partition. I still recommend the fix unless it doesn't work with `${AWS::Partition}`.

Reply:
I am not familiar with CDK.
In [the doc](https://docs.aws.amazon.com/cdk/api/latest/docs/core-readme.html#pseudo-parameters) of CDK, does this section help for this concern?